src/bindings/python/setup.cfg

```diff
@@ -13,6 +13,7 @@ setenv =
   OV_BACKEND = {env:OV_BACKEND:"CPU"}
   PYTHONPATH = {env:PYTHONPATH}
   OpenVINO_DIR = {env:OpenVINO_DIR}
+  CI = True
```

Comment:
do we need it?

Reply:
Yes, openvino-ngraph-onnx-lin jobs run Python tests using tox https://github.com/openvinotoolkit/openvino/blob/137c8e896f1ab36c88378242c022e911890c6c31/.ci/openvino-onnx/Dockerfile#L76

But tox does not pass environmental variables, they need to be specified in config: https://github.com/tox-dev/tox/issues/2659. This was the reason why these jobs didn't find CI or TF_BUILD variable in environment.

Reply:
ok, alternatively we could add:
```
CI = {env:CI}
```
just to bypass value from env. But maybe your choice is even better to completely ignore telemetry from tests.